source/interprocedural_analyses/taint/taintConfiguration.ml

```diff
@@ -728,23 +728,24 @@ module Error = struct
   type t = {
     kind: kind;
     path: PyrePath.t option;
+    location: JsonParsing.Location.t option;
```

Comment:
What are the cases where we don't have a location? Could we fix those?

Reply:
I wanted to migrate errors one by one to make reviewing easier. Is that okay?